custom_components/unifi_access/hub.py

```diff
@@ -594,13 +596,15 @@ def on_message(self, ws: websocket.WebSocketApp, message):  # noqa: C901
                                     "door_name": existing_door.name,
                                     "door_id": door_id,
                                     "actor": actor,
+                                    "authentication": authentication,
                                     "type": ACCESS_EVENT.format(type=access_type),
                                 }
                                 _LOGGER.info(
                                     "Door name %s with id %s accessed by %s. access type: %s",
                                     existing_door.name,
                                     door_id,
                                     actor,
+                                    authentication,
                                     access_type,
                                 )
```

Comment:
```suggestion
                                _LOGGER.info(
                                    "Door name %s with id %s accessed by %s. authentication %s, access type: %s",
                                    existing_door.name,
                                    door_id,
                                    actor,
                                    authentication,
                                    access_type,
                                )
```

Also can you update the README.md to add this new information?